finality-provider/service/fp_instance.go

```diff
@@ -561,48 +513,7 @@ func (fp *FinalityProviderInstance) checkBlockFinalization(ctx context.Context,
 
 // CommitPubRand commits a list of randomness from given start height
 func (fp *FinalityProviderInstance) CommitPubRand(ctx context.Context, startHeight uint64) (*types.TxResponse, error) {
```

Comment:
Do we need this function given that it's 1 line? Same q for `GetLastCommittedHeight`